bgflow/nn/flow/crd_transform/ic.py

```diff
@@ -241,97 +247,48 @@ class ReferenceSystemTransformation(Flow):
         raise warnings if manifold boundaries are violated
     """
 
-    def __init__(self, normalize_angles=True, eps=1e-7, enforce_boundaries=True, raise_warnings=True):
+    def __init__(self, normalize_angles=True, orientation="euler", eps=1e-7, enforce_boundaries=True, raise_warnings=True):
         super().__init__()
         self._normalize_angles = normalize_angles
         self._eps = eps
         self._enforce_boundaries = enforce_boundaries
         self._raise_warnings = raise_warnings
+        assert orientation in ["euler"]
```

Comment:
are there going to be any other methods that you want to support in the future?

Reply:
Initially, I had the version `matrix` as well. It should support the old API (where the orientation was given by the basis frame). However, it was not clear to me how to support this when having multiple return values.